functions/Get-DbaAgentSchedule.ps1

```diff
@@ -0,0 +1,78 @@
+FUNCTION Get-DbaAgentSchedule
```

Comment:
@DevOpsFu so, is it certain that the collection of all jobserver.job.schedules is exactly the same as jobserver.sharedschedules?

@potatoqualitee in that case, it should stay Get-DbaAgentSchedule and I think this PR is fine.
@wsmelton  do you agree?

I also guess we have the complete set of functions on schedules then, not?
We have New- Get- Set- Remove- and no renames are needed; DbaAgentSchedule covers all.
@sanderstad can you confirm that? Do any of your functions need adaptations?